pkg/prometheus/promcfg_test.go

```diff
@@ -5510,6 +5521,263 @@ scrape_configs:
 	}
 }
 
+func TestKeepDroppedTargets(t *testing.T) {
+	expectNoKeepDroppedTargets := `global:
```

Comment:
any special reason to not use golden files here?

Reply:
updated